activesupport/lib/active_support/claims.rb

```diff
@@ -0,0 +1,57 @@
+require 'active_support/time'
+
+module ActiveSupport
+  class Claims
```

Comment:
Given we want Claims to be private - https://github.com/sbhatore/rails/pull/3/files#r34388981 - we need to add a `:nodoc:` so it won't show up in the generated documentation.

Like this:

```ruby
class Claims # :nodoc:
```